### PR TITLE
Added "Integrating depthai into produtcs" section

### PR DESCRIPTION
**Pull request by MOj0**

### CODE REVIEW

File: docs/source/pages/integrating_depthai_into_products.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. _integrating_depthai_into_products:
+
+Integrating DepthAI into products
+=================================
+
+How hard is to integrate DepthAI into our product
```
Comment (Erol444) [inline docs/source/pages/integrating_depthai_into_products.rst:6]: not everything needs to be in Q&A/FAQ style.

File: docs/source/pages/integrating_depthai_into_products.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+*************************************************
+
+Not hard at all, since our platform is designed from the grounds up to achieve simple integration with
+customers products.
+
+What is SoM
+***********
+
+SoM stands for System on Module. In general it is just a small computer used to solve a specific task.
+We have 3 types of SoM devices: `OAK-SoM <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1099.html>`__,
```
Comment (Erol444) [inline docs/source/pages/integrating_depthai_into_products.rst:16]: Make a bullet list of them, mention difference between them

File: docs/source/pages/integrating_depthai_into_products.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+How hard is to integrate DepthAI into our product
+*************************************************
+
+Not hard at all, since our platform is designed from the grounds up to achieve simple integration with
+customers products.
+
+What is SoM
+***********
+
+SoM stands for System on Module. In general it is just a small computer used to solve a specific task.
```
Comment (Erol444) [inline docs/source/pages/integrating_depthai_into_products.rst:15]: small computer? Specify that there's a VPU (Myriad X, quick description about it) + electronics to power it (+ RAM / potentially flash

File: docs/source/pages/integrating_depthai_into_products.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+although USB boot is still available. The main difference between them is NOR flash size
+(OAK-SoM has 16MB, while the other two have 128MB) and boot modes they support, for example
+OAK-SoM-Pro also supports SD-card and Ethernet (EEPROM) boot.
+
+Baseboards
+**********
+
+Just as our `software <https://github.com/luxonis/depthai-python>`__, our
+`hardware <https://github.com/luxonis/depthai-hardware>`__ is opensource too. That way it is not just a black
+box, you can see how everything is made and even change it however you like. Even high-school students
```
Comment (Erol444) [inline docs/source/pages/integrating_depthai_into_products.rst:35]:
```suggestion
box, you can see how our devices (PCB) are designed and you can change them however you like. Even high-school students
```

File: docs/source/pages/integrating_depthai_into_products.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+.. image:: /_static/images/DM1098OBC/DM1098OBC.jpg
+
+NOR Flash and Powering
+**********************
+
+The SoMs have the QSPI NOR Flash, which is capable of quick random access location and is used to
+store and run code. This is the key factor to support the standalone use case.
+
+Power consumption can vary depening on the application. A stereo vision application running Mobilenet-SSD
+V2 at 30 FPS typically consumes about 2.5 W, but more aggressive applications can consume up to 5 W. Most
```
Comment (Erol444) [inline docs/source/pages/integrating_depthai_into_products.rst:54]:
```suggestion
V2 at 30 FPS typically consumes about 2.5 W, but more computationally heavy applications can consume up to 5 W. Most
```

**Review by Luxonis-David — CHANGES_REQUESTED** [review]

**Review by Luxonis-David — APPROVED** [review]
LGTM

File: docs/source/pages/integrating_depthai_into_products.rst [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+.. image:: /_static/images/baseboard.jpeg
+
+And here is the OAK-D-IoT-75, which is a baseboard with on-board cameras and ESP32:
+
+.. image:: /_static/images/DM1098OBC/DM1098OBC.jpg
+
+NOR Flash and Powering
+**********************
+
+The SoMs have the QSPI NOR Flash, which is capable of quick random access location and is used to
```
Comment (Erol444) [inline docs/source/pages/integrating_depthai_into_products.rst:52]: OAK-SOM does not have NOR flash.

File: docs/source/pages/integrating_depthai_into_products.rst [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+Difficulity of integrating DepthAI into products
+************************************************
+
+When designing our platform from the grounds up, we always thought of user integration and made is as simple
+as possible to integrate it into other products. 
+
+What is SoM
+***********
+
+OAK SoM (System on Module) is a small form-factor PCB that features a powerful `Myriad X <https://newsroom.intel.com/wp-content/uploads/sites/11/2017/08/movidius-myriad-xvpu-product-brief.pdf>`__ VPU (Vision Processing Unit) by Intel. The VPU has 16 powerful SHAVE cores and also features the Neural Compute Engine — a dedicated hardware accelerator for deep neural network inference. In addition, all our SoM devices have NOR Flash, which can be used as an alternative to the USB boot. The idea of SoM is that customer can take and build their own device with it, since SoM is very complex, 12 layer PCB. That way our SoM devices serve as an abstraction layer. They can also be used standalone - without host computer, although not all devices support that use-case.
```
Comment (Erol444) [inline docs/source/pages/integrating_depthai_into_products.rst:15]: `In addition, all our SoM devices have NOR Flash,` - see David's answer.

**Review by Erol444 — CHANGES_REQUESTED** [review]

**Review by Erol444 — APPROVED** [review]
Good enough for an initial release, TODO:
- fix typos/spelling mistakes
- info abourt manufacturing process
- Who do we recommend
- approximate cost for 10/100/1000/..
-  sourcing components/cameras etc.

**Pull request merged**